src/bsp/src/stm32/src/SettingsContainer.cpp

```diff
@@ -2,10 +2,12 @@
 #include "DefaultSettings.h"
 #include <cstring>
 
-uint16_t SettingsContainer::GetUUID() { return UUID; }
+uint16_t SettingsContainer::getUUID() { return UUID; }
 
-uint32_t SettingsContainer::GetHostPort() { return HOST_PORT; }
+uint32_t SettingsContainer::getHostPort() { return HOST_PORT; }
 
-uint8_t SettingsContainer::GetHostIP(char* buf, uint8_t length) {
+uint8_t SettingsContainer::getHostIP(char* buf, uint8_t length) {
     return snprintf(buf, length, HOST_IP);
-}
\ No newline at end of file
+}
+
+LogLevel SettingsContainer::getLogLevel() { return LOG_LEVEL; }
```

Comment:
In the ROS version we allow `Err` and `Warning` while here we don't. Maybe remove the extra supported strings from the ROS version